e2e-tests/constants/field-ids/insurance/exporter-business/index.js

```diff
@@ -25,5 +25,6 @@ export const EXPORTER_BUSINESS = {
   TURNOVER: {
     FINANCIAL_YEAR_END_DATE: 'financialYearEndDate',
     ESTIMATED_ANNUAL_TURNOVER: 'estimatedAnnualTurnover',
+    PERCENTAGE_TURNOVER: 'percentageTurnover',
```

Comment:
Since the description of this field is "Percentage of turnover from exports", shall we call this

`exportsTurnoverPercentage`?